lib/sablon/parser/mail_merge.rb

```diff
@@ -1,34 +1,37 @@
+# frozen_string_literal: true
+
 module Sablon
   module Parser
     class MailMerge
       class MergeField
-        KEY_PATTERN = /^\s*MERGEFIELD\s+([^ ]+)\s+\\\*\s+MERGEFORMAT\s*$/
+        KEY_PATTERN = /^\s*MERGEFIELD\s+(.+?)\s+\\\*\s+MERGEFORMAT\s*$/.freeze
```

Comment:
Previous expression captured exactly one word without spaces in it. The new if-statement format uses spaces to separate operands and the operator, so instead we are now capturing everything between `MERGEFIELD` and the first `\*`.